src/item/views/modals/AddFragmentToCollection.tsx

```diff
@@ -22,44 +24,48 @@ import {
 	ToolbarRight,
 } from '@viaa/avo2-components';
 import { Avo } from '@viaa/avo2-types';
-import { ApolloQueryResult } from 'apollo-client';
-import { get } from 'lodash-es';
+
 import {
 	GET_COLLECTION_BY_ID,
 	GET_COLLECTION_TITLES_BY_OWNER,
 	INSERT_COLLECTION,
 	INSERT_COLLECTION_FRAGMENT,
 } from '../../../collection/graphql';
 import { DataQueryComponent } from '../../../shared/components/DataComponent/DataQueryComponent';
+import { FlowPlayer } from '../../../shared/components/FlowPlayer/FlowPlayer';
 import {
 	formatDurationHoursMinutesSeconds,
 	toSeconds,
 } from '../../../shared/helpers/formatters/duration';
 import { dataService } from '../../../shared/services/data-service';
+import { fetchPlayerToken } from '../../../shared/services/player-service';
 import toastService, { TOAST_TYPE } from '../../../shared/services/toast-service';
 import './AddFragmentToCollection.scss';
 
 interface AddFragmentToCollectionProps {
 	externalId: string;
-	itemInfo: Avo.Item.Response;
+	itemMetaData: Avo.Item.Response;
 	isOpen: boolean;
 	onClose: () => void;
 }
 
 export const AddFragmentToCollection: FunctionComponent<AddFragmentToCollectionProps> = ({
 	externalId,
-	itemInfo,
+	itemMetaData,
 	isOpen,
 	onClose = () => {},
 }) => {
+	const [playerToken, setPlayerToken] = useState();
 	const [createNewCollection, setCreateNewCollection] = useState(false);
 	const [selectedCollectionId, setSelectedCollectionId] = useState<string>('');
 	const [selectedCollection, setSelectedCollection] = useState<Avo.Collection.Response | undefined>(
 		undefined
 	);
 	const [newCollectionTitle, setNewCollectionTitle] = useState('');
 	const [fragmentStartTime, setFragmentStartTime] = useState<number>(0);
-	const [fragmentEndTime, setFragmentEndTime] = useState<number>(toSeconds(itemInfo.duration) || 0);
+	const [fragmentEndTime, setFragmentEndTime] = useState<number>(
+		toSeconds(itemMetaData.duration) || 0
```

Comment:
default toSeconds niet naar 0?

Reply:
Typescript complains als ik de `|| 0` er niet bij voeg.

Reply:
inderdaad, het kan nog steeds null returnen als het een string is maar een verkeerd formaat heeft en silent = true
dus de `|| 0` is nodig, my bad